app/models/review.rb

```diff
@@ -0,0 +1,4 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  blongs_to :book
```

Comment:
This may or may not have been causing you  problems, but you have belongs_to spelled wrong here.